docs/sources/mimir/configure/configure-kafka-backend.md

```diff
@@ -0,0 +1,35 @@
+---
+aliases:
+  - ../operators-guide/configure/configure-kafka-backend/
+description: Learn how to configure Grafana Mimir to use Kafka for ingest storage.
+menuTitle: Kafka
+title: Configure Grafana Mimir kafka backend
+weight: 66
+---
+
+# Configure the Grafana Mimir Kafka backend
```

Comment:
@tacole02 I think this doesn't recognize Kafka as a product name. Do you know where we need to make a change for the linter to expect "Kafka" and not "kafka"?

Reply:
That's a good question, @dimitarvdimitrov , I'll defer to @jdbaldry.

Reply:
i'll merge this now because it's still not going to the `main` branch and I need the change. But we should figure this out before we merge to `main` (or go with "kafka" instead of "Kafka")